features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -645,13 +640,20 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
 
         setSubmitting(true);
 
-        const promises: Promise<void | ValidationDataInterface[]>[] = [ updatePasswordPolicies ];
-
-        if (isPasswordInputValidationEnabled) {
-            promises.push(updateValidationConfigData(processedFormValues, null, validationData[0]));
-        }
-
-        Promise.all(promises)
+        serverConfigurationConfig
```

Comment:
Let's add a comment saying that this is a temporary workaround.

Reply:
Even with the backend fix, I believe keeping sequential API calls is still beneficial. The backend validation API logic itself makes further calls to the governance configuration. So, it's better to ensure that governance changes are fully persisted before triggering the validation API.

Reply:
Added a comment.